### PR TITLE
Action: Get Unused Indexes in PostgreSQL

### PR DESCRIPTION
**Pull request by shloka-bhalgat-unskript**
### Testing
Screenshot 2023-03-15 at 5 22 37 PM

### Checklist:
- [x] My changes generate no new warnings.
- [x] I have added tests that prove my fix is effective or that my feature works.
- [x] Any dependent changes have been merged and published. 

### Documentation
Make sure that you have documented corresponding changes in this repository. 

[EN-4246]: https://unskript.atlassian.net/browse/EN-4246?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by dougsillars — DISMISSED** [review]

File: Postgresql/legos/postgresql_check_unused_indexes/__init__.py [hunk 1]
```diff
@@ -0,0 +1,6 @@
+#
```
Comment (harshal-netserv) [inline Postgresql/legos/postgresql_check_unused_indexes/__init__.py:1]: This file should be empty @shloka-bhalgat-unskript

File: Postgresql/legos/postgresql_check_unused_indexes/postgresql_check_unused_indexes.py [hunk 2]
```diff
@@ -0,0 +1,85 @@
...
+            "index": records[1],
+            "index_size": records[2],
+            "index_scans": records[3],
+        }
+        output.append(result)
+        data.append([records[0], records[1], records[2], records[3]])
+
+    if len(res) > 0:
+        headers = ["table", "index", "index_size", "index_scans"]
+        output = tabulate(data, headers=headers, tablefmt="grid")
```
Comment (amit-chandak-unskript) [inline Postgresql/legos/postgresql_check_unused_indexes/postgresql_check_unused_indexes.py:77]: Again same issue as the other PR, we shouldnt be returning tabulate output from a check. It should return data. @shloka-bhalgat-unskript

**Review by amit-chandak-unskript — CHANGES_REQUESTED** [review]
Take care of comments

File: Postgresql/legos/postgresql_check_unused_indexes/postgresql_check_unused_indexes.py (hunk 2 above)
Comment (shloka-bhalgat-unskript) [reply Postgresql/legos/postgresql_check_unused_indexes/postgresql_check_unused_indexes.py]: Fixed @amit-chandak-unskript

File: Postgresql/legos/postgresql_check_unused_indexes/__init__.py (hunk 1 above)
Comment (shloka-bhalgat-unskript) [reply Postgresql/legos/postgresql_check_unused_indexes/__init__.py]: Changed @harshal-netserv

**Review by amit-chandak-unskript — APPROVED** [review]

**Pull request merged**